client/apps/edit/components/content2/sections/header/controls/VideoControls.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { SectionVideo } from '../../video/index.jsx'
+
+export default class VideoControls extends Component {
+  static propTypes = {
+    article: PropTypes.object.isRequired,
+    isOpen: PropTypes.bool.isRequired,
+    onClick: PropTypes.func.isRequired
+  }
+
+  render () {
+    const { article, isOpen, onClick } = this.props
+
+    return (
+      <div className='edit-header--video'>
+        <div
+          onClick={onClick}
+          className='edit-header--video-open'
+        >
+          Embed Video
+        </div>
+        <div>
+          {isOpen &&
+            <div
+              className='edit-section-container'
+              data-editing
+              data-type='video'
+            >
+              <SectionVideo editing
+                article={article}
+                section={article.heroSection}
+                channel={{ isArtsyChannel: () => false }} // TODO: is this needed?
+                headerType={'basic-embed'}
```

Comment:
The channel prop is for the backbone channel, which has isArtsyChannel as a method on the model.

Reply:
Can you elaborate on what an artsy "channel" is? Isn't everything technically a channel since content can't exist outside of one?

Reply:
A channel is, for lack or a better term, the owner of the article -- for example 'artsy editorial', or 'artsy auctions' -- different channels have different features and permissions available to them.